config/default.yml

```
@@ -875,6 +875,10 @@ Rails/TimeZoneAssignment:
     - spec/**/*.rb
     - test/**/*.rb
 
+Rails/TransactionExitStatement:
+  Description: 'Avoid the usage of `return`, `break` and `throw` in transaction blocks.'
```

Can you insert `Enabled: pending`?

```suggestion
  Description: 'Avoid the usage of `return`, `break` and `throw` in transaction blocks.'
  Enabled: pending
```